recipes-openxt/initrdscripts/initramfs-xenclient/init.sh

```diff
@@ -27,6 +27,26 @@ ROOT_READONLY=
 DEFINIT=/sbin/init
 FIRSTBOOT_FLAG=/boot/system/firstboot
 
+is_tpm_2_0 () {
+    [ -e /sys/class/tpm/tpm0/device/description ] && cat /sys/class/tpm/tpm0/device/description | grep "2.0" &>/dev/null
```

Comment:
Right now our kernel doesn't expose as much through the sysfs as the 1.2 TPM did.  "caps" doesn't exist on 2.0.  "description" doesn't seem to exist on 1.2.  Seems to me that the absence of one is enough to confirm the tpm version.  In the event that "description" shows up on both, grepping for 2.0 should differentiate between the versions.